extensions/geode-modules/src/main/java/org/apache/geode/modules/session/catalina/ClientServerSessionCache.java

```diff
@@ -225,6 +221,17 @@ void createSessionRegionOnServers() {
     }
   }
 
+  Region<String, HttpSession> createLocalSessionRegionWithRegisterInterest() {
+    Region<String, HttpSession> region = createLocalSessionRegion();
+
+    // register interest are needed for proxy or caching-proxy client:
```

Comment:
It will be nice to call out the requirements separately.
In case of proxy session region Interest registration is done to get the destroy callbacks and for caching-proxy to replicate the data across clients.